extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -179,6 +179,13 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
      * that it has been verified.
      */
     if (this.isResponsive_()) {
+      if (!this.element.hasAttribute('data-full-width')) {
+        user().error(TAG,
```

Comment:
Same idea here

Reply:
Here you would look at this.getViewport().getSize().width == this.getIntersectionElementLayoutBox().width